src/levelui.cpp

```diff
@@ -782,10 +807,23 @@ void Level::RenderUI()
 						ImGui::Checkbox("Show Wireframe", &GuiRenderSettings::showWireframe);
 						ImGui::Checkbox("Show Backfaces", &GuiRenderSettings::showBackfaces);
 						ImGui::Checkbox("Show Level Verts", &GuiRenderSettings::showLevVerts);
-						ImGui::Checkbox("(NOT IMPL) Show Checkpoints", &showCheckpoints);
-						ImGui::Checkbox("(NOT IMPL) Show Starting Positions", &showStartingPositions);
-						ImGui::Checkbox("(NOT IMPL) Show BSP Rect Tree", &showBspRectTree);
+						ImGui::Checkbox("Show Checkpoints", &GuiRenderSettings::showCheckpoints);
+						ImGui::Checkbox("Show Starting Positions", &GuiRenderSettings::showStartpoints);
+						ImGui::Checkbox("Show BSP Rect Tree", &GuiRenderSettings::showBspRectTree);
 						ImGui::PushItemWidth(textFieldWidth);
+						{
+							int temp = GuiRenderSettings::bspTreeTopDepth, temp2 = GuiRenderSettings::bspTreeBottomDepth;
+							ImGui::SliderInt("BSP Rect Tree top depth", &GuiRenderSettings::bspTreeTopDepth, 0, GuiRenderSettings::bspTreeMaxDepth);
```

Comment:
`SliderInt` will return true upon new values, no need to store temp values and compare with them afterwards